Source/Core/Core/HW/SI/SI_DeviceGCAdapter.cpp

```diff
@@ -72,7 +72,10 @@ int CSIDevice_GCAdapter::RunBuffer(u8* buffer, int length)
 bool CSIDevice_GCAdapter::GetData(u32& hi, u32& low)
 {
   CSIDevice_GCController::GetData(hi, low);
```

Comment:
Feels a bit weird that this is called _before_ the `DeviceConnected` check...should this be here?